WinUIGallery/ControlPages/DesignGuidance/IconographyPage.xaml

```diff
@@ -182,22 +182,18 @@
                     BorderThickness="1,0,0,0"
                     DataContext="{x:Bind}">
                     <StackPanel>
-                        <TextBlock
```

Comment:
And I'd remove this, so we start with the icon. and then all the properties that can be copied.